jax/experimental/jax2tf/jax2tf.py

```diff
@@ -54,6 +53,8 @@
 from jax._src import linear_util as lu
 from jax._src import op_shardings
 from jax._src import sharding_impls
+from jax._src import maps
```

Comment:
Can this be `from jax._src.maps import xmap, xmap_p`?

Reply:
Do we want `xmap_p` to be in the top-level `jax2tf` namespace? I lean toward no.

Reply:
Okay, sounds good. Not a big deal. This was just to make xmap deletion easier but someone will have to do that work anyways ;)